README.md

```diff
@@ -1 +1,45 @@
-# broadway-sensitive-data
+broadway/sensitive-data
+=======================
+
+Helpers for handling sensitive data with Broadway.
+
+[![Build Status](https://travis-ci.org/broadway/broadway-sensitive-data.svg?branch=master)](https://travis-ci.org/broadway/broadway/broadway-sensitive-data)
+
+## Installation
+
+```
+$ composer require broadway/broadway-sensitive-data
+```
```

Comment:
I think it would be good to add a small usage section. I think it's helpful to explain how to register the processors (and to not register them with the regular event listener.